docs/developer-guide/test-e2e.md

```diff
@@ -13,6 +13,16 @@ Git repository via file url: `file:///tmp/argocd-e2e***`.
 
 You can observe the tests by using the UI [http://localhost:8080/applications](http://localhost:8080/applications) with username `"admin"` and password `"password"`.
 
+### Running Single E2E Test
+
+The `test-e2e` target passes the `TEST_FLAGS` environment variable to the `go test` run inside the local ArgoCD server container. To run a single e2e test, set this environment variable execution: `TEST_FLAGS="-run ^TestCustomToolWithEnv\$$" make test-e2e`
+
+_Dollar signs need to be escaped with a double dollar for the `Makefile` forwarding the execution to the underlying container._
```

Comment:
I think you could just use single quotes in that example for the TEST_FLAGS value to avoid escaping dollar signs.

Reply:
in order to pass it in single quotes, it needs to be `TEST_FLAGS='-run ^TestCustomToolWithEnv$$' make test-e2e`